Sources/NIO/SocketAddresses.swift

```diff
@@ -126,12 +126,12 @@ public enum SocketAddress: CustomStringConvertible {
     }
 
     /// Get the port associated with the address, if defined.
-    public var port: UInt16? {
+    public var port: Int? {
         switch self {
         case .v4(let addr):
-            return UInt16(bigEndian: addr.address.sin_port)
+            return Int(UInt16(bigEndian: addr.address.sin_port))
```

Comment:
Why is this round-tripping through `UInt16`?

Reply:
@Lukasa step 1) change the endianness (only available within the same type), step 2) change the type

Reply:
Can you add a comment to that effect please? People will definitely forget.

Reply:
yes, done